app/templates/data.hbs

```diff
@@ -24,7 +24,7 @@
             {{#each layerGroup.meta.url as |url|}}
               <span style="display:block;" class="no-margin">
                 <a href="{{url}}" target="_blank">
-                  {{fa-icon "external-link-alt"}}
+                  {{fa-icon icon="external-link-alt"}}
```

Comment:
@allthesignals  is there a reason you changed some of these to the new `<FaIcon />` syntax and not others? Strictly speaking, switching over the syntax is just a nice-to-have for your work if it's not blocking upgrades, but I'd prefer to be consistent if we're going to do it.